aten/src/ATen/TensorSubclassLikeUtils.h

```diff
@@ -44,4 +44,10 @@ inline bool areAnyTensorSubclassLike(TensorList tensors) {
   return std::any_of(tensors.begin(), tensors.end(), isTensorSubclassLike);
 }
 
+inline bool areAnyOptionalTensorSubclassLike(const torch::List<c10::optional<Tensor>>& tensors) {
```

Comment:
nit: c10::List ? There is actually a reason why we might not want to use torch::List -- the fb internal build system might treat libtorch differently, I don't remember